opencsp/app/scene_reconstruction/test/test_SceneReconstruction.py

```diff
@@ -76,19 +80,31 @@ def setUpClass(cls, dir_input: str = None, dir_output: str = None):
         scene_recon.align_points(marker_ids, alignment_values, verbose=verbose)
 
         # Test results
-        cls.pts_exp = np.loadtxt(
-            join(dir_output, 'point_locations.csv'), delimiter=',', skiprows=1
-        )
         cls.pts_meas = scene_recon.get_data()
+        cls.scene_recon = scene_recon
+        cls.dir_output = dir_output
 
     def test_calibrated_corner_locations(self):
         """Tests relative corner locations"""
-        np.testing.assert_allclose(self.pts_meas, self.pts_exp, atol=1e-5, rtol=0)
+        pts_exp = np.loadtxt(
+            join(self.dir_output, 'point_locations.csv'), delimiter=',', skiprows=1
+        )
+        np.testing.assert_allclose(self.pts_meas, pts_exp, atol=1e-5, rtol=0)
         print('Corner locations tested successfully.')
 
+    def save_csv(self):
+        """Saves CSV file of points to data location"""
+        base_dir = os.path.dirname(__file__)
+        file = join(base_dir, 'data/data_expected/point_locations.csv')
+        self.scene_recon.save_data_as_csv(file)
```

Comment:
This is cool. I didn't know about numpy's CSV handling capabilities.

Alternatively, and I'm DEFINITELY not saying to do this now, just wanted you to be aware of the `common/lib/file/CsvInterface.py` class in case it is ever helpful.  The intention is for each row in a CSV file to be represented by an object, and for said row to be able to serialize/deserialize itself from the file.